docs-source/content/faq/node-heating.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Node heating problem"
+date: 2020-06-03T08:08:19-04:00
+draft: false
+weight: 22
+---
+
+The WebLogic Operator creates a Pod for each WebLogic Server instance that is started. The [Kubernetes Scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/) then selects a Node for each Pod. Because the default scheduling algorithm gives substantial weight to selecting a Node where the necessary Docker images have already been pulled, this often results in Kubernetes running many of the Pods for WebLogic Server instances on the same Node while other Nodes are not fairly utilized.
```

Comment:
WebLogic Operator -> WebLogic Server Kubernetes Operator (when we use operator capitalized, I prefer that we use the product name, either WebLogic Server Kubernetes Operator or Oracle WebLogic Server Kubernetes Operator. Please make this change globally, if you want to use the full product name, or use lowercase operator.)